src/k8s_autoscale/main.py

```diff
@@ -55,8 +55,8 @@ def adjust_scale(api, target_replicas, deployment_namespace, deployment_name):
 
 
 @retriable(sleeptime=3, max_sleeptime=10, retry_exceptions=(TaskclusterRestFailure,))
-def get_pending(queue, provisioner, worker_type):
-    return queue.pendingTasks(provisioner, worker_type)["pendingTasks"]
```

Comment:
It's possible that `worker_type` is now the taskQueueId (essentially `f"{provisioner}/{worker_type}"` concatenated), and it's possible we're now doing queries for nonexistent `worker_type`s now.

Do we know if the `cfg["worker_type"]` is a full `taskQueueId` or just the second half of one?